packages/web/src/components/collection/desktop/PublishButton.tsx

```diff
@@ -13,62 +9,42 @@ import { Tooltip } from 'components/tooltip'
 import { PublishConfirmationModal } from './PublishConfirmationModal'
 
 const { getCollection } = collectionPageSelectors
-const { getCollecitonHasHiddenTracks } = cacheCollectionsSelectors
 
 const messages = {
   publish: 'Make Public',
   publishing: 'Making Public',
-  emptyPlaylistTooltipText: 'You must add at least 1 song.',
-  hiddenTracksTooltipText: (collectionType: 'playlist' | 'album') =>
-    `You cannot make a ${collectionType} with hidden tracks public.`
+  emptyPlaylistTooltipText: 'You must add at least 1 song.'
 }
 
-type PublishButtonProps = Partial<ButtonProps> & {
+type PublishButtonProps = Partial<IconButtonProps> & {
   collectionId: number
 }
 
 export const PublishButton = (props: PublishButtonProps) => {
   const { collectionId, ...other } = props
-  const { _is_publishing, track_count, is_album } = useSelector(
-    (state: CommonState) => getCollection(state, { id: collectionId })
+  const { _is_publishing, track_count } = useSelector((state: CommonState) =>
+    getCollection(state, { id: collectionId })
   ) as Collection
-  const hasHiddenTracks = useSelector((state: CommonState) =>
-    getCollecitonHasHiddenTracks(state, { id: collectionId })
-  )
 
   const [isConfirming, toggleIsConfirming] = useToggle(false)
 
-  const isDisabled = !track_count || track_count === 0 || hasHiddenTracks
+  const isDisabled = !track_count || track_count === 0
```

Comment:
do we still want this to be disabled for playlists?

I'm in favor of getting rid of special cases, but should check w/product especially if this won't open the modal for playlists.

Reply:
Hmm, actually in hindsight I'm gonna pull these changes back till we hear more about the intended design from product